hooks/proprietary/PSMDST.js

```diff
@@ -17,22 +17,26 @@
 'use strict'
 
 /*
-0  1  2  3
-|  |  |  |
-$PSMDST,xx,yy,nn*CS
+0  1  2  3  4
+|  |  |  |  |
+$PSMDST,Z,xx,yy,nn*CS
 where:
 PSMDST     	Raymarine Seatalk1 datagram sentence
-0 			00-9C       	Datagram type
-1 			hex       	First datagram content
-2 			hex   		Last datagram content
-3 			hex      	Checksum
+0       C/R       R for Recevied messages, C for sent messages *Note: This field only exists in later firmware versions of the ShipModul Miniplex
```

Comment:
typo in _Recevied_.

Could you please line the field number markers up with the fields like so
<img width="202" alt="image" src="https://user-images.githubusercontent.com/1049678/217063247-a23f4208-5d28-479f-9f77-640e50e8a154.png">